gtsfm/scene_optimizer.py

```diff
@@ -347,6 +359,9 @@ def aggregate_frontend_metrics(
     success_count_unit3 = np.sum(unit3_errors < angular_err_threshold_rad)
     success_count_pose = np.sum(pose_errors < angular_err_threshold_rad)
 
+    # count entries with all correct correspondences
+    all_correct = [1 for x in corr_correctness if x[1] == 1.0]
```

Comment:
is `np.sum()` faster?

Reply:
I have to convert it to a numpy array anyway?

Reply:
maybe if the numpy array can use a memcpy from the list array that is faster than a for-loop in the list comprehension